src/Sulu/Bundle/ContactBundle/Api/ContactAddress.php

```diff
@@ -91,8 +91,6 @@ public function setAddress(AddressEntity $address)
      */
     public function getAddress()
     {
-        $adr = $this->entity->getAddress();
-
-        return new AddressEntity($adr);
+        return $this->entity->getAddress();
```

Comment:
same here there is no api entity for Address and new AddressEntity does not make sense here.